packages/idealo.design/scripts/motifuiImporter.js

```diff
@@ -1,25 +1,17 @@
-const path= require('path')
-
-const motifUiFolder = path.resolve(__dirname,'./../src/server/motif-ui-components/')
-const pathToMotifUiRepo = path.resolve(__dirname,'../../../../motif-ui/src')
-
-const fs = require('fs');
-const fse = require("fs-extra")
-const http = require('http')
+const path = require('path')
+const fse = require('fs-extra')
+const fs = require('fs')
+const FormData = require('form-data')
+const axios = require('axios')
+
+const motifUiFolder = path.resolve(__dirname, './../src/server/motif-ui-components/')
+const pathToMotifUiRepo = path.resolve(__dirname, '../../../../motif-ui/src')
+const localScreenshots = path.resolve(__dirname, '../src/server/screenshots')
```

Comment:
The server directory is off limits for the importer. Treat the server as its own application, it's outside and normally inaccessible from the importer app. If a path needs to be constructed to this location then it's the server that should be doing it.